vignettes/LPWC.Rmd

```diff
@@ -245,6 +245,7 @@ dist <- 1 - LPWC::corr.bestlag(simdata[11:20, ], timepoints = timepoints, max.la
 cutree(hclust(dist), k = 3)
 ```
 
+## Filtering genes
```

Comment:
This is the last piece needed before merging the pull request.  Can you please write a couple sentences to explain that LPWC works best when run with genes that are significantly differentially expressed over time and genes that do not vary are not allowed?  The user must filter them before running LPWC.  You can give a code snippet to show users how to filter 0 variance genes.